data/base-theurgy.yaml

```diff
@@ -1,4 +1,48 @@
 ---
+immortal_to_aspect:
```

Comment:
A similar mapping is done in `commune.lic`: https://github.com/rpherbig/dr-scripts/blob/master/commune.lic#L147-L159. Is there a benefit to updating that script, too?

Reply:
Done; haven't tested this yet, though.